recipes/petsird-python/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "petsird" %}
```

Comment:
```suggestion
{% set python_min = python_min|default("3.8") %}
{% set name = "petsird" %}
```

Reply:
@ocefpaf the pypi package says `python >= 3.9`. Can we also have `{% set python_min = python_min|default("3.9") %}` ?

Reply:
Yep. You should adjust that to what is in the package metadata.